packages/script-loader/package.json

```diff
@@ -42,6 +42,10 @@
     "test": "vitest -c ../../configs/vitest.config.ts",
     "test:ssr": "pnpm run test --mode ssr"
   },
+  "devDependencies": {
+    "@types/jsdom": "^21.1.1",
```

Comment:
Those should probably be added to the root package.json do they are available in each package
`jsdom` is already added this way

Reply:
I know, but I get a TS error, since it cannot find the dependencies in the local package.json.

Reply:
I mean it has to be just the types, they are not added to root package.json. Might as well at it there. It should be available just as `vitest` it

Reply:
Let me try tomorrow.